services/workflow/workflow-service/src/main/java/com/milesight/beaveriot/rule/components/webhook/WebhookComponent.java

```diff
@@ -0,0 +1,59 @@
+package com.milesight.beaveriot.rule.components.webhook;
+
+import com.milesight.beaveriot.rule.annotations.RuleNode;
+import com.milesight.beaveriot.rule.api.ProcessorNode;
+import com.milesight.beaveriot.rule.constants.RuleNodeType;
+import com.milesight.beaveriot.rule.util.SecureUtil;
+import lombok.Data;
+import org.apache.camel.Exchange;
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.spi.UriParam;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Map;
+import java.util.Random;
+
+/**
+ * @author loong
+ * @date 2024/12/17 15:56
+ */
+@RuleNode(value = "webhook", type = RuleNodeType.EXTERNAL, description = "Webhook")
+@Data
+public class WebhookComponent implements ProcessorNode<Exchange> {
+
+    @UriParam
+    private Map<String, String> inputArguments;
+    @UriParam
+    private Map<String, Object> headers;
+    @UriParam
+    private String webhookUrl;
+    @UriParam
+    private String secretKey;
+
+    @Autowired
+    private ProducerTemplate producerTemplate;
+
+    @Override
+    public void processor(Exchange exchange) {
+        if (headers != null) {
+            exchange.getIn().setHeaders(headers);
+        }
+        if(inputArguments != null) {
+            exchange.getIn().setBody(inputArguments);
```

Comment:
The argument to inputArguments may be an expression

Reply:
Fixed